examples/dreambooth/train_dreambooth.py

```diff
@@ -35,6 +35,13 @@ def parse_args():
         required=True,
         help="Path to pretrained model or model identifier from huggingface.co/models.",
     )
+    parser.add_argument(
+        "--pretrained_model_name_revision",
```

Comment:
wonder whether we should call this just `revision` @patil-suraj what do you think?

Reply:
Agree, let's just call this `revision`